py/torch_tensorrt/dynamo/conversion/plugins/_generate_plugin.py

```diff
@@ -0,0 +1,176 @@
+import tensorrt_bindings.plugin as trtp
+from torch._dynamo.source import LocalSource
+import torch
+from torch.fx.experimental.symbolic_shapes import DimDynamic, ShapeEnv
+from sympy import lambdify
+from typing import Tuple
+from types import FunctionType
+from torch._subclasses.fake_tensor import FakeTensorMode
+from torch.fx.experimental.symbolic_shapes import ShapeEnv
+from sympy import lambdify
+
+def mksym(shape_env, value, source, dynamic_dim):
+    return shape_env.create_symintnode(
+        shape_env.create_symbol(
+            value,
+            source=source,
+            dynamic_dim=dynamic_dim,
+        ),
+        hint=value,
+        source=source,
+    )
+
+def generate_plugin(plugin_name : str):
+    namespace, name = plugin_name.split("::")
+    
+    # retrieve the corresponding torch operation using the passed in string
+    torch_op = getattr(getattr(torch.ops, namespace), name)
+    
+    # helper function that generates the required signature based on the torch operation
+    def generate_signature(torch_op):
+        schema = torch_op._schemas['']
+        tensor_args = []
+        arg_list = []
+        
+        args = []
+        kwargs = []
+        
+        register_func_annotation = {}
+        impl_func_annotation = {}
+        
+        for arg in schema.arguments:
+            arg_list.append(arg.name)
+            
+            if arg.type.isSubtypeOf(torch._C.TensorType.get()):
+                tensor_args.append(arg)
+                register_func_annotation[arg.name] = trtp.TensorDesc
+                impl_func_annotation[arg.name] = trtp.Tensor
+            elif arg.type.isSubtypeOf(torch._C.FloatType.get()):
+                register_func_annotation[arg.name] = float
+                impl_func_annotation[arg.name] = float
+            elif arg.type.isSubtypeOf(torch._C.IntType.get()):
+                register_func_annotation[arg.name] = int
+                impl_func_annotation[arg.name] = int
+            else:
+                raise ValueError("arg type is not handled")
+                
```

Comment:
What other arg types would we need or is this it?

Reply:
@narendasan Other types are needed as well, this is just a temporary handling.

Reply:
Do you have a list?